fip-x.md

```diff
@@ -0,0 +1,115 @@
+---
+fip: X
+title: FIO Domain Marketplace
+status: Draft
+type: Functionality
+author: Isaiah Williams - EOS BlockSmith
+updated: 2/5/21
```

Comment:
Please add created [as per template](https://github.com/fioprotocol/fips/blob/master/fip-TEMPLATE.md) and use YYY-MM-DD format to avoid confusion.